message-service/src/main/kotlin/fi/espoo/evaka/msg/properties/SfiSoapProperties.kt

```diff
@@ -32,7 +32,10 @@ data class SfiPrintingProperties(
     var forcePrintForElectronicUser: Boolean = false,
     var printingProvider: String? = "Edita",
     var billingId: String? = null,
-    var billingPassword: String? = null
+    var billingPassword: String? = null,
+    var contactPersonName: String? = "",
+    var contactPersonPhone: String? = "",
+    var contactPersonEmail: String? = ""
```

Comment:
There is a related infra pull request where the following environment variables are defined: `FI_ESPOO_EVAKA_MSG_SFI_PRINTING_CONTACTPERSONNAME`, `FI_ESPOO_EVAKA_MSG_SFI_PRINTING_CONTACTPERSONPHONE`, `FI_ESPOO_EVAKA_MSG_SFI_PRINTING_CONTACTPERSONEMAIL`. If I understand correctly, Spring will infer these from there(??)

Reply:
https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.external-config.typesafe-configuration-properties.relaxed-binding

I.e. you are correct in your assumption (though you can also always test that locally).